src/query-helpers.js

```diff
@@ -64,6 +82,21 @@ function makeGetAllQuery(allQuery, getMissingError) {
         container,
       )
     }
+    if (getConfig().showSuggestions) {
+      //get a unique list of all suggestion messages.  We are only going to make a suggestion if
```

Comment:
@kentcdodds you missed one. 🤣 
```suggestion
      // get a unique list of all suggestion messages.  We are only going to make a suggestion if
```